src/sentry/api/endpoints/project_repo_path_parsing.py

```diff
@@ -56,8 +58,9 @@ def org_id(self):
     def validate_source_url(self, source_url: str):
         # first check to see if we are even looking at the same file
         stack_path = self.initial_data["stack_path"]
-        stack_file = stack_path.split("/")[-1]
-        source_file = source_url.split("/")[-1]
+
+        stack_file = PureWindowsPath(stack_path).name
+        source_file = PureWindowsPath(source_url).name
```

Comment:
sorry, nitpicking again... if the `source_url` is an actual URI, than proper way would be to 

```python
PurePath(urlparse(source_url).path).name
```

Reply:
Good call.